src/Modules/AuthModule/components/Login/Login.tsx

```diff
@@ -44,101 +55,137 @@ export default function Login() {
       saveAdminData && saveAdminData(); // Call saveAdminData only if it exists
     } catch (error) {
       if (axios.isAxiosError(error) && error.response) {
-        showErrorToast(error.response.data.message)}
-        else {
-          // Handle other types of errors here
-          showErrorToast("An error occurred.");
-        }
+        showErrorToast(error.response.data.message);
+      } else {
+        // Handle other types of errors here
+        showErrorToast("An error occurred.");
+      }
     } finally {
       setSpinner(false);
     }
   };
 
   return (
-    <div className="Auth-container vh-100 row align-items-center justify-content-center overflow-auto gx-0 flex-nowrap ">
-      <div className="logo  col-md-5 text-center">
-        <img src={logo} alt="logo" className="mb-3" />
-      </div>
+    <>
+      <section className="">
+        <div className="bg-login vh-100 text-white d-flex  align-items-center  ">
+          <div className="container  d-flex justify-content-center ">
+            <div className="row  w-100">
+              <div className="col-md-6 mx-auto text-center ">
+                {/* register */}
+                <div className="login w-100 ">
+                  <img className="mb-3" src={Images.authLogo} alt="" />
+                  {/* form */}
+                  <div className="form  rounded-4 py-5">
+                    <div className="container-auth-form pt-4  ">
+                      <div className="form-title position-relative ">
+                        <p>welcome to PMS</p>
+                        <h3 className="text-main">Login</h3>
+                      </div>
+                      {/* form body */}
+                      <div className="form-body mt-3">
+                        {/* */}
+                        <form onSubmit={handleSubmit(onSubmit)}>
+                          {/* email input */}
+                          <div className="input-container">
+                            <input
+                              placeholder="Enter your E-mail"
+                              className={`input-field ${
+                                errors.email && "border-danger "
+                              }`}
+                              type="text"
+                              {...register("email", {
```

Comment:
We need to update this to use the validations created in the utils.

Reply:
we'll be working on it after the merge, to avoid more conflict